src/services/user.ts

```diff
@@ -0,0 +1,22 @@
+import { WithValidationError } from "../types/withValidationError"
```

Comment:
As [mentioned](https://autoricardo-ag.slack.com/archives/C0289PXUV8A/p1633096739000300?thread_ts=1632823022.003600&cid=C0289PXUV8A) by Slavko, the User endpoints are currently part of the dealer service, what's the thought behind putting them into a separate file and not the one handling the endpoints for the dealer service?

Reply:
I missed that and since it is under Users I created a new file